tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -990,6 +990,48 @@ def test_single_sign_on_ldap_ipa_server(enroll_idm_and_configure_external_auth,
         )
 
 
+@destructive
+def test_single_sign_on_ldap_ad_server(enroll_ad_and_configure_external_auth):
+    """Verify the single sign-on functionality with external authentication
+
+    :id: 3c233aa4-c817-11ea-b105-d46d6dd3b5b2
+
+    :setup: Enroll the AD Configuration for External Authentication
+
+    :steps: Assert single sign-on session user is directed to satellite instead of login page
+
+    :expectedresults: After single sign on, user should be redirected from /extlogin to /hosts page
```

Comment:
This test doesn't seem to check that. Please extend test, or update expected results.

Reply:
updated as we can only test using curl from sat-host